IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers/WolfSSLDemo/src/wolfDemo/wolf_main.c

```diff
@@ -0,0 +1,253 @@
+/* wolf_main.c
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+#include <string.h>
+#include <wolf_main.h>
+#include "r_sys_time_rx_if.h"
+#include "r_cmt_rx_if.h"
+#include "Pin.h"
+#include "r_t4_itcpip.h"
+
+#define SSL_SERVER
+#define ETHER_TASK
+extern const UB _t4_dhcp_enable;
+static int dhcp_accept_flg = 0;
+static UW tcpudp_work[14800] = {0};
+UB	dhcp_evt = 0;
+void dhcp_check(void);
+void print_dhcp(VP param);
+#define DHCP_ENABEL        (1)
+#define IF_CH_NUMBER       (0)
+
+void main(void) 
+{
+    /*Toopers start function */
+    startw();
+}
+
+void timer_int_Wrapper() 
+{
+#ifndef ETHER_TASK
+    cmt1_isr();
+#endif
+ }
+
+ void timer_cm2_int_Wrapper() 
+ {
+    int du = 0;
+    timeTick((void *)&du);
+ }
+extern bool		sns_ctx(void);
+
+void ether_int_Wrapper() 
+{
+    R_BSP_InterruptControl(BSP_INT_SRC_AL1_EDMAC0_EINT0, BSP_INT_CMD_CALL_CALLBACK, FIT_NO_PTR);
+}
+void taskEther(intptr_t exinf) 
+{
+    int dhcp_cnt = 0;
+    bool flg = false;
+    while(1) {
+     if (flg == false) {
+       waisem_ether_wrapper();
+       flg = true;
+     }
+     dhcp_check();
+     dly_tsk(1);
+    }
+}
+/* DHCP Reset Event Check */
+void dhcp_check(void)
+{
+
+    if(DHCP_ENABEL == _t4_dhcp_enable && dhcp_evt == DHCP_EV_PLEASE_RESET) {
+     tcpudp_reset(IF_CH_NUMBER);
+     dhcp_evt = 0;
+    }
+    return;
+}
+
+void taskNetWork(intptr_t exinf) 
+{
+    UB ctl = 0;
+    UW i = 0;
+    while(1) {
+     if(!(i % 10)) {
+      ctl = ctl ? 0 : 1;
+     }
+     LED_CTL(ctl)
+     i++;
+#ifdef ETHER_TASK
+     cmt1_isr();
+#endif
+     dly_tsk(1);
+    }
+
+}
+#define FREQ 10000 /* Hz */
+extern void timeTick(void *pdata);
+
+void taskDemoWolf(intptr_t exinf) 
+{
+    uint32_t channel;
+    wolfSSL_init();
+    if(!init_ether()) return ;
+    R_CMT_CreatePeriodic(FREQ, &timeTick, &channel);
+    ICU.SLIBXR128.BYTE = 1; //select B
+
+    sigsem_ether_wrapper();
+    while(1) {
+     dly_tsk(10);
```

Comment:
Fix indent.

Reply:
Correction of review remarks .
Fixed indent.